.github/workflows/main.yml

```diff
@@ -36,3 +36,5 @@ jobs:
           echo "REPO_SITE=${owner}" >> $GITHUB_ENV
       - name: Build with Gradle
         run: ./gradlew build --warning-mode all
+      - name: Spotless Check
+        run: ./gradlew spotlessCheck
```

Comment:
Doesn't this already happen with `build`? Or should we change `build` to `assemble`?

Reply:
You are right. The build consists of spotlessCheck. I have updated the PR with separate tasks. If you think use `./gradlew build` is clearer, we may close this PR.